### PR TITLE
[VG-8580] Fix screen's functions label

### PR DESCRIPTION
**Pull request by mgrattepanche-ledger**
### What is it about ? 

- Labels were not correctly conjugate. 
- Put "send" before "receive"

### CODE REVIEW

**Review by slundy-ledger — COMMENTED** [review]
"Minimum amount to be received" -> will this be displayed on the PSD ? i am afraid it won't fit the screen

File: ethereum/paraswap/parsers.json [hunk 1]
```diff
@@ -184,12 +184,12 @@
...
                         {
-                            "label": "Received minimum amount",
-                            "name": "toAmount"
+                            "label": "Amount to send",
+                            "name": "fromAmount"
                         },
                         {
-                            "label": "Sent amount",
-                            "name": "fromAmount"
+                            "label": "Minimum amount to be received",
```
Comment (pcollet-ledger) [inline ethereum/paraswap/parsers.json:191]: This does not match what we have on the figma right?
https://www.figma.com/file/BK1aCdRYEoRrfcmgtH5Xd1/Smart-Contract-Interactions-%26-dApp-Browser?node-id=646%3A7231

File: ethereum/paraswap/parsers.json [hunk 2]
```diff
@@ -248,12 +248,12 @@
...
```
Comment (pcollet-ledger) [inline ethereum/paraswap/parsers.json:255]: This as well?

**Review by pcollet-ledger — CHANGES_REQUESTED** [review]

**Comment by mgrattepanche-ledger** [comment]
Done through https://github.com/LedgerHQ/ledger-asset-dapps/pull/55/files I guess ... ?

**Pull request closed without merging**